src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubPullRequestFilter.java

```diff
@@ -26,17 +26,19 @@
 
 import hudson.Extension;
 import hudson.model.Descriptor;
+import hudson.model.TopLevelItem;
+import hudson.model.View;
 import hudson.views.ViewJobFilter;
-import jenkins.branch.Branch;
-import jenkins.branch.SimpleViewBranchFilter;
+import java.util.List;
+import jenkins.scm.api.SCMHead;
 import org.kohsuke.stapler.DataBoundConstructor;
 
 /**
  * A {@link ViewJobFilter} that matches {@link PullRequestSCMHead} based branches.
  *
  * @since FIXME
  */
-public class GitHubPullRequestFilter extends SimpleViewBranchFilter {
+public class GitHubPullRequestFilter extends ViewJobFilter {
```

Comment:
Seems too low-level to be defining `ViewJobFilter`s in a branch source. I would have expected this to be handled in `branch-api` via some better abstracted API, so that clients like Blue Ocean would be able to present the same information in very different UIs, without reference to the item hierarchy.